README.md

```diff
@@ -63,51 +38,34 @@ $ ./flockeropenssl/generate_flocker_certs.sh -h
 -h | --help (This help message)
 ```
 
-Example
+Examples:
 
 ```
-./flockeropenssl/generate_flocker_certs.sh -d="ec2-52-91-11-106.compute-1.amazonaws.com" -n="ec2-52-91-11-106.compute-1.amazonaws.com,node2,node3" -f=/etc/flocker/ssl/flockeropenssl/openssl.cnf
+./flockeropenssl/generate_flocker_certs.sh -d=www.foobar.com -k=1024 -c=staging-1 -n=one,two
 ```
-
-Example Output
 ```
-0:ec2-52-91-11-106.compute-1.amazonaws.com
-1:node2
-2:node3
-Generating RSA private key, 4096 bit long modulus
-.....................................................................++
-....................................++
-e is 65537 (0x10001)
-Generating RSA private key, 4096 bit long modulus
-..............................................................................++
-.
-.
-. (cut out long output)
+./flockeropenssl/generate_flocker_certs.sh -d="ec2-52-91-11-106.compute-1.amazonaws.com" -n="ec2-52-91-11-106.compute-1.amazonaws.com,node2,node3" -f=/etc/flocker/ssl/flockeropenssl/openssl.cnf
 ```
 
-Your SSL directory should be populated with the new certificates
+All relevant certificates can be found in `clusters/<cluster_name>`
+
+
+### How to use the certificates?
+
+#### Control node
+
 ```
-$ ls
-api_user.crt  cluster.key                                           csr                                       index.txt           index.txt.old  node3
-api_user.key  control-ec2-52-91-11-106.compute-1.amazonaws.com.crt  ec2-52-91-11-106.compute-1.amazonaws.com  index.txt.attr      newcerts       serial
-cluster.crt   control-ec2-52-91-11-106.compute-1.amazonaws.com.key  flockeropenssl                            index.txt.attr.old  node2          serial.old
+$ scp cluster/cluster-1/cluster.crt user@cluster-master:/etc/flocker/
+$ scp cluster/cluster-1/control-service.* user@cluster-master:/etc/flocker/
 ```
 
-How to use the certificates?
+#### Node
+
 ```
-# Copy control, cluster and API certs
-$ cp *.crt *.key /etc/flocker/
-$ cd /etc/flocker/
-
-# Rename them appropriately 
-$ mv api_user.crt plugin.crt
-$ mv api_user.key plugin.key
-$ mv control-ec2-52-91-11-106.compute-1.amazonaws.com.crt control-service.crt
-$ mv control-ec2-52-91-11-106.compute-1.amazonaws.com.key control-service.key
-
-# On the dataset node, copy the appropriate node certs (you may need to scp these to appropriate locations)
-$ cp ssl/node2/node-da0779a7-51b9-4d62-a4a7-e9ca55f73988.crt node.crt
-$ cp ssl/node2/node-da0779a7-51b9-4d62-a4a7-e9ca55f73988.key node.key
+$ scp cluster/cluster-1/cluster.crt user@cluster-master:/etc/flocker/
```

Comment:
should these saw `cluster-node` instead of `cluster-master` for the node example?